tests/transactions/setup-governance-updater.ts

```diff
@@ -0,0 +1,58 @@
+import {
+    TransactionFunctionParams,
+    sendTransaction,
+    FlowConstants,
+  } from '../../utils/flow'
+  
+  const CODE = (address: string) => `
+  import AxelarGovernanceService from ${address}
+
+    transaction(blockUpdateBoundary: UInt64, contractName: String, code: String) {
+        prepare(signer: AuthAccount) {
+            if !signer.getCapability<&AuthAccount>(AxelarGovernanceService.UpdaterContractAccountPrivatePath).check() {
+                signer.unlink(AxelarGovernanceService.UpdaterContractAccountPrivatePath)
+                signer.linkAccount(AxelarGovernanceService.UpdaterContractAccountPrivatePath)
+            }
+            let accountCap: Capability<&AuthAccount> = signer.getCapability<&AuthAccount>(AxelarGovernanceService.UpdaterContractAccountPrivatePath)
+            if signer.type(at: AxelarGovernanceService.UpdaterStoragePath) != nil {
+                panic("Updater already configured at expected path!")
+            }
+            signer.save(
+                <- AxelarGovernanceService.createNewUpdater(
```

Comment:
I belive this needs to be updated for the new Updater interface which houses a Host Capability

Reply:
I think this file was a remnant from a refactor.  was replaced by publish-auth-capability.ts.  ill go ahead and remove it!